testobserve/tests/build.gradle

```
@@ -0,0 +1,94 @@
+/*
+ * Copyright (c) 2022, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+apply plugin: 'java'
+
+configurations {
+    jbalTools
```

We should drop the transitive dependencies
```suggestion
    jbalTools {
        transitive false
    }
```

---

Fixed with https://github.com/ballerina-platform/module-ballerinai-observe/pull/1/commits/147f0fbaaaf1de6478d08699f0c0968553e50a5c